apps/condo/domains/ticket/access/Ticket.js

```diff
@@ -111,6 +115,30 @@ async function canManageTickets (args) {
             return ticket.client === user.id
         }
     } else {
+        if (isBulkRequest && operation === 'create') return false
+
+        if (isBulkRequest && operation === 'update') {
+            if (itemIds.length !== uniq(itemIds).length) return false
+            if (itemIds.length !== originalInput.length) return false
+
+            if (!originalInput.every((updateItem) => {
+                for (const key in updateItem.data) {
+                    if (!BULK_UPDATE_ALLOWED_FIELDS.includes(key)) return false
+                }
+                return true
+            })) return false
```

Comment:
do we really need this?

Reply:
Now, with a single update, we have a check of the address for its compliance with the organization. With a bulk update, we must also support this logic. However, the task of updating and maintaining all possible scenarios of bulk operations is a different task. Therefore, we decided to temporarily limit the update to only two fields - essotor and asizjnee.